Source/ScrollingNavigationController.swift

```diff
@@ -8,6 +8,10 @@ import UIKit
      Called when the state of the navigation bar changes
      */
     optional func scrollingNavigationController(controller: ScrollingNavigationController, didChangeState state: NavigationBarState)
+
+    /*******joaoritter*******/
+    optional func scrollingNavigationController(controller: ScrollingNavigationController, willChangeState state: NavigationBarState)
```

Comment:
Don't worry about that, but please  remove the ****** comments and use proper comments describing the func as the one above it.